classes/Gems/Handlers/Setup/RoleHandlerTrait.php

```diff
@@ -32,19 +33,24 @@ protected function getUsedPrivileges()
                 $privilegeNames[$privilege] = implode("<br/>&nbsp; + ", $labels);
             }
 
-            foreach ($privileges as $privilege) {
-                if (!isset($privilegeNames[$privilege])) {
-                    $privilegeNames[$privilege] = $privilege;
+            asort($privilegeNames);
+
+            asort($privileges);
+
+            foreach ($privileges as $resourceName => $label) {
+                if (!isset($privilegeNames[$resourceName])) {
+                    $privilegeNames[$resourceName] = $label;
```

Comment:
Dus we sorteren eerst `$privilegeNames`, en voegen daarna in alfabetische volgorde de `$privileges` er aan toe?

Reply:
Correct. Zo krijg je ipv alles alfabetisch gesorteerd eerst de menu rollen, dan de api rollen, dan de rest routes die geen menu item hebben, dan de rollen die losstaan van routes.